helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/ConstraintBasedAlgorithm.java

```diff
@@ -42,6 +42,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static java.lang.Float.compare;
```

Comment:
nit, better directly call Float.compare in the code. Static import here is not quite desired.

Reply:
TFTR. Updated.